apps/api/src/controllers/session.controller.js

```diff
@@ -0,0 +1,31 @@
+import bcryptjs from "bcryptjs";
+import jwt from "jsonwebtoken";
+import { findOne } from "../operations/users";
+
+async function login(req, res, next) {
+  try {
+    const user = await findOne(req.body.username);
+    if (user) {
+      const result = await bcryptjs.compare(req.body.password, user.password);
+      if (result === true) {
```

Comment:
I think that can be redundant, probably we can change it to
```suggestion
      if (result) {
```

If `result` its a boolean then we can go with my suggestion